### PR TITLE
adding in the ability to set the contrast on the ssd1306

### PR DESCRIPTION
**Pull request by lgleasain**
The ssd1306 allows you to set the contrast on a display with 256 unique steps. This allows you to set it.

### CODE REVIEW

File: ssd1306/src/main/java/com/google/android/things/contrib/driver/ssd1306/Ssd1306.java [hunk 1]
```diff
@@ -238,6 +239,19 @@ public void setPixel(int x, int y, boolean on) throws IllegalArgumentException {
...
     /**
+     * Sets the contrast for the display.
+     *
+     * @param level The contrast level (0-255).
+     */
+    public void setContrast(int level) throws IOException, IllegalArgumentException {
+        if (level < 0 || level > 255) {
+            throw new IllegalArgumentException("contrast out of bound:" + level);
+        }
+        mI2cDevice.writeRegByte(0, (byte) COMMAND_CONTRAST_LEVEL );
```
Comment (jdkoren) [inline ssd1306/src/main/java/com/google/android/things/contrib/driver/ssd1306/Ssd1306.java]: nit: remove space before closing parenthesis (same for line below)

File: ssd1306/src/main/java/com/google/android/things/contrib/driver/ssd1306/Ssd1306.java [hunk 2]
```diff
@@ -238,6 +239,19 @@ public void setPixel(int x, int y, boolean on) throws IllegalArgumentException {
     }
 
     /**
+     * Sets the contrast for the display.
+     *
+     * @param level The contrast level (0-255).
+     */
+    public void setContrast(int level) throws IOException, IllegalArgumentException {
+        if (level < 0 || level > 255) {
+            throw new IllegalArgumentException("contrast out of bound:" + level);
```
Comment (jdkoren) [inline ssd1306/src/main/java/com/google/android/things/contrib/driver/ssd1306/Ssd1306.java]: Let's put the acceptable range in the exception message, e.g. "Contrast level must be between 0 and 255"

**Comment by lgleasain** [comment]
With the unit tests, would it not be more appropriate to add them in a test file under the ssd1306 driver rather than one for the BMX280 which is a different component? Right now there are not any tests for this driver but I'd be happy to start that.

**Comment by jdkoren** [comment]
@lgleasain you are right, my mistake.

**Comment by lgleasain** [comment]
added in unit tests, improved the message in the exception for an invalid value and removed un-neccessary spaces.

**Comment by mangini** [comment]
Thanks for the contribution! One minor issue: can you check if mI2cDevice is null like the other methods?

**Comment by lgleasain** [comment]
added a check for it along with a unit test.

**Pull request merged**